dojo/templates/dojo/snippets/comments.html

```diff
@@ -30,10 +30,14 @@ <h4>Notes <span class="pull-right"><a data-toggle="collapse" href="#vuln_notes">
                 </button>
             </form>
         </div>
-    {% endifequal %}
+    {% endif %}
   </div>
   <div class="panel-body">
-    {{ note }}
+    {% if note.note_type != None %}
+      <strong>Note type : {{ note.note_type }}</strong>
+      <br><br>
+    {% endif %}
+    {{ note.entry }}
```

Comment:
Why did we have to use "note.entry" instead of "note"?

Reply:
Can go with "note". Done.